vips/vip-245.ts

```diff
@@ -0,0 +1,235 @@
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../src/types";
+import { makeProposal } from "../src/utils";
+
+export const blocksPerYear = 10512000;
+export const blocksPerDay = parseInt((blocksPerYear / 365).toString());
+
+export const UNITROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const VAI_VAULT_RATE = parseUnits("62.50", 18).div(blocksPerDay);
```

Comment:
This is `2170138888888888`. If we multiply `2170138888888888 * 28800` we get `62499999999999974400`, that would be shown in the UI this way: `62.49`

It we round up, adding +1, we would have `2170138888888889`. If we multiply `2170138888888889 * 28800` we get `62500000000000003200`, that would be shown in the UI this way: `62.50`. So, I think I would round up this number, to improve their aspect in the UI and taking into account it's a minimal change in the value